src/openforms/payments/contrib/demo/plugin.py

```diff
@@ -19,12 +23,21 @@ def handle_return(self, request, payment):
         payment.status = PaymentStatus.completed
         payment.save()
 
+        token = submission_status_token_generator.make_token(payment.submission)
+        status_url = request.build_absolute_uri(
+            reverse(
+                "api:submission-status",
+                kwargs={"uuid": payment.submission.uuid, "token": token},
+            )
+        )
+
         form_url = payment.submission.cleaned_form_url
```

Comment:
is there any more work needed to make this work properly with hash-based routing too? Using https://github.com/open-formulieren/open-forms/blob/master/src/openforms/frontend/frontend.py#L9

Reply:
I am not sure, I will look into it!

Reply:
I added the support for it!